src/components/prep/ProductionRunDetailDialog.tsx

```
@@ -0,0 +1,319 @@
+  completed: { label: 'Completed', variant: 'default', tone: 'bg-emerald-100 text-emerald-800 border-emerald-200' },
+  cancelled: { label: 'Cancelled', variant: 'outline', tone: 'text-muted-foreground' },
+  draft: { label: 'Draft', variant: 'outline', tone: 'text-muted-foreground' },
+};
+
+export function ProductionRunDetailDialog({ run, open, onOpenChange, onSave, saving }: ProductionRunDetailDialogProps) {
+  // Guard clause: don't render if no run data - must be before any hooks
+  if (!run) {
+    return null;
+  }
+
+  const [actualYield, setActualYield] = useState<number | ''>('');
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)